File: python/account.py
import random;

class Account: 
  id = int
  name = str
  document = str
  email = str
  password = str
  
  def __init__(self, name, document, email, password):
    self.id = random.randint(0, 100000)
    self.name = name
    self.document = document
    self.email = email
    self.password = password

File: js/Account.js
class Account {
  id;
  name;
  document;
  email;
  password;

  constructor(name, document, email, password){
    this.id = new Date().getDate();
    this.name = name;
    this.document = document;
    this.email = email;
    this.password = password;
  }
}

class Driver extends Account {
  constructor(name, document, email, password) {
    super(name, document, email, password)
  }
}

class User extends Account {
  constructor(name, document, email, password) {
    super(name, document, email, password)
  }
}